app/controllers/articles_controller.rb

```diff
@@ -11,7 +11,24 @@ def new
 
   def index
     @page_title = "Home | Haxxor News"
-    @articles = Article.order(id: :desc)
+    if params[:article_options]
+      case params[:article_options]
+        when "Most Recent"
+        @articles = Article.order(id: :desc)
+        when "All-Time Top Rated"
+        @articles = article_records(Vote.article_top_10)
+        when "Yearly Top Rated"
+        @articles = article_records(Vote.article_top_10_year)
+        when "Monthly Top Rated"
+        @articles = article_records(Vote.article_top_10_month)
+        when "Daily Top Rated"
+        @articles = article_records(Vote.article_top_10_day)
```

Comment:
i think this would read a little nicer if you defined scopes on the `Article` model class for these guys